packages/components/scrollbar/src/bar.vue

```diff
@@ -29,6 +29,11 @@ const handleScroll = (wrap: HTMLDivElement) => {
   }
 }
 
+onDeactivated(() => {
+  moveX.value = 0
+  moveY.value = 0
+})
+
```

Comment:
Thank you for your contribution!

## my opinion

When a user uses keep-alive, the user would expect to have the previous state intact. But the change doesn't seem to keep the previous state by initializing moveX, moveY to 0.

Reply:
You're right. I'll modify the code ASAP.